### PR TITLE
chore(search): unnest styles for react

### PR DESCRIPTION
**Pull request by aledavila**
Unnest styles. Weren't showing up in react.

Related: https://github.com/IBM/carbon-components-react/pull/2005

### CODE REVIEW

**Review by dakahn — APPROVED** [review]
🐦 Great!

**Review by asudoh — APPROVED** [review]
LGTM basically, though wrt nesting - Did you set `true` to `CARBON_USE_EXPERIMENTAL_FEATURES` as well as to `CARBON_USE_BREAKING_CHANGES` environment variables? I can see below in carbon-components-react#2005:

image

Edit: If you set `true` to `CARBON_REACT_STORYBOOK_USE_EXTERNAL_CSS` as well as to `CARBON_REACT_STORYBOOK_USE_STYLE_SOURCEMAP` in addition to above, you can navigate to the source Sass code, too:

css-debugging-react

Thanks @aledavila!

File: src/components/search/_search.scss [hunk 1]
```diff
@@ -279,15 +279,20 @@
...
+    background-color: $hover-ui;
+    border-bottom: 1px solid $ui-04;
+  }
+  .#{$prefix}--search-close:active {
+    @include focus-outline('outline');
+    background-color: $selected-ui;
+  }
+
+  .#{$prefix}--search--xl .#{$prefix}--search-close svg {
+    width: 20px;
```
Comment (joshblack) [inline src/components/search/_search.scss]: Should we use `rem` or `mini-units` helpers for this? Also, is the size for the icon to be a 20x20 icon? If so, we might want to just use the 20x20 icon itself.

**Review by joshblack — CHANGES_REQUESTED** [review]

File: src/components/search/_search.scss (hunk 1 above)
Comment (aledavila) [reply src/components/search/_search.scss]: I removed it and will add the icon in react

**Comment by aledavila** [comment]
Fixed in react

**Pull request closed without merging**